### PR TITLE
Add Tensorflow YoloV3 MLOps project

### PR DESCRIPTION
**Pull request by alexandergg**
Add Tensorflow YoloV3 MLOps project

### CODE REVIEW

File: examples/MLOps-Tensorflow-YoloV3/.gitignore [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
```
Comment (jpe316) [inline examples/MLOps-Tensorflow-YoloV3/.gitignore]: Don’t need the .gitignore file

**Comment by jpe316** [comment]
@alexandergg would you rather use a submodule?

File: examples/MLOps-Tensorflow-YoloV3/.gitignore (hunk 1 above)
Comment (alexandergg) [reply examples/MLOps-Tensorflow-YoloV3/.gitignore]: Delete .gitignore https://github.com/microsoft/MLOps/pull/44/commits/28207efd057e5518f98c33ec885fd5cab21e0221

**Comment by alexandergg** [comment]
> @alexandergg would you rather use a submodule?

@jpe316 What do you mean with submodules ??

**Pull request merged**